src/colorize.cr

```diff
@@ -167,6 +167,29 @@ module Colorize
     def back(io : IO) : Nil
       (to_i + 10).to_s io
     end
+
+    def self.from_lscolors?(color : String) : Color?
```

Comment:
Why not using `Char` for this? Would just fit perfectly. It's faster, easier to understand. `String` just doesn't make any sense here. `Char` says exactly what is wanted here: one character.

Reply:
`Char` does make more sense, I'll make that change.